webapp/channels/src/actions/marketplace.ts

```diff
@@ -23,9 +23,9 @@ import type {GlobalState} from 'types/store';
 
 import {doAppSubmit, openAppsModal, postEphemeralCallResponseForContext} from './apps';
 
-export function fetchRemoteListing(): ActionFunc {
-    return async (dispatch: DispatchFunc, getState: GetStateFunc) => {
-        const state = getState() as GlobalState;
+export function fetchRemoteListing(): NewActionFuncAsync<MarketplacePlugin[], GlobalState> { // HARRISONTODO unused
```

Comment:
should we default to always GlobalState of web rather than GlobalState of MM redux

Reply:
We can't really do that without the web app having two different versions of `ActionFunc` for the web app and for mattermost-redux. I've considered doing that, but I didn't bother changing that yet